bson/primitive/objectid_test.go

```diff
@@ -233,3 +233,20 @@ func TestObjectID_UnmarshalJSON(t *testing.T) {
 		})
 	}
 }
+
+func TestObjectID_MarshalText(t *testing.T) {
```

Comment:
Seems good to add these tests, but where did they come from?

Reply:
Great sounds good, thanks for investigating/adding this.

Reply:
Fantastic, the new linters are already proving helpful!